tests/unit_tests/include/test_helper.hpp

```diff
@@ -119,6 +145,30 @@
 #define CHECK_HOST_OR_CPU(q) q.is_host() || q.get_device().is_cpu()
 #endif
 
+#define TEST_RUN_CT_SELECT_NO_ARGS(q, func)                                \
```

Comment:
If it's not required for other domains except DFT I think this macro can be simplified to only backends supported by DFT.

Reply:
Done in [f7566e1](https://github.com/oneapi-src/oneMKL/pull/261/commits/f7566e176c1cb7b30c5b8757acb43c2483fe3967).